### PR TITLE
Liftr Confluent generated commands based out of stable version 13-02-2024

### PR DESCRIPTION
**Pull request by DeepikaNMS**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by DeepikaNMS** [comment]
PR is based out of the swagger version - https://github.com/Azure/azure-rest-api-specs/pull/27756

**Comment by jsntcy** [comment]
@DeepikaNMS, If you want to release the changes in this PR immediately in a new version, you need update the version and history notes as well.

**Comment by DeepikaNMS** [comment]
@jsntcy: I have updated the history file and the version. Please review 
There is a PR for aaz repo https://github.com/Azure/aaz/pull/420

File: src/confluent/azext_confluent/azext_metadata.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
     "azext.isExperimental": false,
```
Comment (necusjz) [inline src/confluent/azext_confluent/azext_metadata.json:2]:
```suggestion
```
`azext.isExperimental` is deprecated, please refer to https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#backward-compatibility-of-extension-version-tags

**Comment by necusjz** [comment]
please update `setup.py`

File: src/confluent/azext_confluent/azext_metadata.json (hunk 1 above)
Comment (DeepikaNMS) [reply src/confluent/azext_confluent/azext_metadata.json]: I have changed this

Comment (necusjz) [reply src/confluent/azext_confluent/azext_metadata.json]: yes, but
> For all stable extension releases, neither azext.isPreview nor azext.isExperimental can be added into module metadata.

**Review by necusjz — APPROVED** [review]

**Pull request merged**